src/frontend/sharedComponents/icons/Circle.js

```diff
@@ -4,16 +4,26 @@ import { StyleSheet, View } from "react-native";
 import type { ViewStyleProp } from "../../types";
 
 type CircleProps = {
+  color?: string,
+  borderWidth?: number,
   radius?: number,
   children: React.Node,
   style?: ViewStyleProp,
 };
 
-const Circle = ({ radius = 25, style, children }: CircleProps) => (
+const Circle = ({
+  color = "#EAEAEA",
+  borderWidth = 2,
+  radius = 25,
+  style,
+  children,
+}: CircleProps) => (
   <View
     style={[
       styles.circle,
       {
+        borderWidth: color === "#EAEAEA" ? null : borderWidth,
+        borderColor: color,
```

Comment:
small likelihood and relatively minor, but this is prone to a bug where if the color is defined in the preset as `#EAEAEA`, the border width will be set to null as opposed to `borderWidth`. Would suggest removing the default assignment in the `color` prop and updating this part to something like this:

```suggestion
        borderWidth: color ? borderWidth : null,
        borderColor: color || `#EAEAEA`,
```

Reply:
Additionally, may be worth validating that `color` is valid using `validate-color` (which is already used elsewhere in the project) and set the border color based on that e.g.

```js
borderColor: color && validateColor(color) ? color : `#EAEAEA`
```